src/node/createSetupServer.ts

```diff
@@ -29,6 +29,12 @@ export function createSetupServer(...interceptors: Interceptor[]) {
    * Sets up a server-side requests interception with the given mock definition.
    */
   return function setupServer(...requestHandlers: RequestHandlersList) {
+    requestHandlers.forEach((handler) => {
+      if (Array.isArray(handler))
+        throw new Error(
+          `[MSW] setupServer function receive every handler as an arg. You should call it as setupServer(...requestHandlers) with requestHandlers the array of handlers.`,
```

Comment:
- please don’t use contractions (args)
- we should include the wrong usage example, presumably how the user tried to call this function.
- we shouldn’t assume the user knows about the `requestHandlers` list (the name is rather internal)

```
[MSW] Failed to call “setupServer”: received a list of request handlers (setupServer([a, b])), expected to receive each handler individually: setupServer(a, b).